tests/integration/components/tag-post-list-test.js

```diff
@@ -1,37 +1,40 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { render, waitFor } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import percySnapshot from '@percy/ember';
-import { TAG } from '../data/tags';
-import { MORE_THAN_3_POSTS, POST } from '../data/posts';
+
+let store;
 
 module('Integration | Component | tag-post-list', function(hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function() {
     this.owner.setupRouter();
+    store = this.owner.lookup('service:store');
   });
 
   test('it renders in Percy', async function(assert) {
-    this.set('tag', TAG);
+    let tag = await store.findRecord('tag', 'getting-started');
+    this.set('tag', tag);
 
-    await render(hbs`<TagPostList @tag={{tag}} />`);
+    await render(hbs`<TagPostList @tag={{this.tag}} />`);
```

Comment:
Passing in `this.tag` here makes sense to me, but I noticed that in many of the other calls to render the component  in the other tests we're passing in tag/tags without the this. My habit at this point is to make an explicit this call even if we don't need it (but I suspect at some point we will).

Reply:
yea I had to change how this whole thing worked because it was using POJOs and needed to use true ember-data models, I thought I would have caught most places where this was wrong but I guess if there are some inconsistencies then they will be caught on an upgrade (or in the beta of ember-try) before it would cause any problems 👍